core/deployment/src/main/java/io/quarkus/deployment/recording/BytecodeRecorderImpl.java

```diff
@@ -256,8 +256,6 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
                         return null;
                     }
                     ProxyInstance instance = getProxyInstance(returnType);
-                    if (instance == null)
-                        return null;
```

Comment:
`getProxyInstance(returnType);` never returns `null`.

Reply:
I'm not sold on this one.

Reply:
Unless I'm missing something, the only value that can be returned from `getProxyInstance(returnType);` comes from https://github.com/quarkusio/quarkus/blob/master/core/deployment/src/main/java/io/quarkus/deployment/recording/BytecodeRecorderImpl.java#L312-L313

Reply:
It's not about missing something. It's just that doesn't cost anything to have it there so I would keep it.

Reply:
It's back with curly braces in bonus!